src/test/test-utils.tsx

```diff
@@ -46,6 +47,14 @@ const customRender = (ui: ReactElement, options?: CustomRenderOptions): Promise<
     history.push(options.path);
   }
 
+  if (!options?.hasLayout) {
+    jest.mock('../parts/Layout', () => {
+      const MockedLayout: React.FC = ({ children }: any) => children;
+      MockedLayout.displayName = 'MockedLayout';
+      return MockedLayout;
```

Comment:
I think it would be good to log to console something like 'Rendering with mocked out `Layout` component' to make sure we know that full dApp is not being rendered (in case there is a test that relies on Layout somehow)